src/app.jsx

```diff
@@ -185,6 +188,7 @@ export const Application = () => {
 
     const _createDirectory = () => createDirectory(Dialogs, currentDir, selectedContext || selected);
     const _createLink = () => createLink(Dialogs, currentDir, files, selectedContext);
+    const _downloadFile = () => downloadFile(currentDir, selectedContext);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-175-20240126-132259-b04878fd-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)